spec/javascript/components/supplemental_application/sections/__snapshots__/RentalAssistance.test.js.snap

```diff
@@ -718,9 +716,7 @@ Array [
             >
               <td
                 className=""
-              >
-                Other
```

Comment:
I wonder why this updated 🤔

Reply:
The snapshot tests weren't very complete at this time - I've updated them a lot so check out the new snapshots and see what you think.